tracing-log/src/lib.rs

```diff
@@ -44,6 +44,34 @@ use tracing_core::{
     Event, Metadata,
 };
 
+static LOGGER: LogTracer = LogTracer { _p: () };
+
+/// Sets up `LogTracer` as global logger for the `log` crate,
+/// with the given level as max level filter.
+///
+/// Setting a global logger can only be done once.
+///
+/// The `level` parameter controls which levels will be passed
+/// to `tracing`.
+pub fn init_logger_with_filter(level: log::LevelFilter) -> Result<(), log::SetLoggerError> {
```

Comment:
TIOLI: What do you think of making this a function on `LogTracer` (moving it to an `impl LogTracer` block), and dropping the word `logger` from the name (so it's a little less wordy), like `LogTracer::init()`/`LogTracer::init_with_filter()`?

This is, admittedly, closer to what other crates (like `env_logger`) do, though...

Reply:
I think it makes sense (especially as this crates handles conversion on both ways, so a global function maybe be confusing, event with log in its name.). I'll add a mention in the crate docs as it's very likely what people using this crate are looking for.

Reply:
Once you add those docs, I'm happy to merge this.